crates/ensure_no_std/src/main.rs

```diff
@@ -0,0 +1,100 @@
+#![feature(start, libc, lang_items)]
+#![feature(alloc_error_handler)]
+#![no_std]
+#![no_main]
+
+// The libc crate allows importing functions from C.
+extern crate libc;
+use core::{
+    alloc::{GlobalAlloc, Layout},
+    cell::UnsafeCell,
+    panic::PanicInfo,
+    ptr::null_mut,
+    sync::atomic::{AtomicUsize, Ordering::SeqCst},
+};
+
+// A list of C functions that are being imported
+extern "C" {
+    pub fn printf(format: *const u8, ...) -> i32;
+}
+
+use nostr;
+// use url_no_std;
+// use rand;
```

Comment:
Are these still needed?

Reply:
The are not, removed in https://github.com/integritee-network/nostr/pull/1/commits/1925dae6668812b41ef5c8b0556b9b39ae4944ea.